card/mod/02_basic_types/set/type/pointer.rb

```diff
@@ -218,6 +218,14 @@ def default_item_view
   end
 end
 
+format :rss do
```

Comment:
I found that rss rendering in pointer cards does not contain cards details as search type does. I am not sure whether we want to do it like search type.